src/app/payments/send-lightning.ts

```diff
@@ -724,7 +723,7 @@ const executePaymentViaLn = async ({
     }
 
     if (!rawRoute) {
-      const reimbursed = await Wallets.reimburseFee({
+      const reimbursed = await reimburseFee({
```

Comment:
This breaks tracing.

Reply:
it's a use case calling another use case. not sure we intend to trace those calls?
tracing from the use case are meant to be called from the graphql layer?